src/docchecks.jl

```diff
@@ -331,8 +342,8 @@ end
 
 GITHUB_ERROR_ADVICE = (
     "This means automatically finding the source URL link for this package failed. " *
-    "Please add the source URL link manually to the `remotes` field " *
-    "in `makedocs` or install the package using `Pkg.develop()``."
+        "Please add the source URL link manually to the `remotes` field " *
+        "in `makedocs` or install the package using `Pkg.develop()``."
 )
```

Comment:
I guess I'm not a huge fan of this one.. but I supposed it's consistent with the general rules?

I wonder if this would be a better general pattern (not for the formatter, but when trying to write long single-line string literals):

```
GITHUB_ERROR_ADVICE = """
    This means automatically finding the source URL link for this package failed. \
    Please add the source URL link manually to the `remotes` field \
    in `makedocs` or install the package using `Pkg.develop()``."""
```

Reply:
Yea it just falls out naturally from the "rule set", but I have grown to like it because it makes it easier to see that this is just one thing and not a three-tuple, compare
```
GITHUB_ERROR_ADVICE = (
    "This means automatically finding the source URL link for this package failed. " *
    "Please add the source URL link manually to the `remotes` field " *
    "in `makedocs` or install the package using `Pkg.develop()``."
)
```
vs
```
GITHUB_ERROR_ADVICE = (
    "This means automatically finding the source URL link for this package failed. ",
    "Please add the source URL link manually to the `remotes` field ",
    "in `makedocs` or install the package using `Pkg.develop()``."
)
```
vs
```
GITHUB_ERROR_ADVICE = (
    "This means automatically finding the source URL link for this package failed. " *
        "Please add the source URL link manually to the `remotes` field " *
        "in `makedocs` or install the package using `Pkg.develop()``."
)
```

Slightly more useful if there are more things though:
```
GITHUB_ERROR_ADVICE = (
    "This means automatically finding the source URL link for this package failed. " *
        "Please add the source URL link manually to the `remotes` field " *
        "in `makedocs` or install the package using `Pkg.develop()``.",
    "This means automatically finding the source URL link for this package failed. " *
        "Please add the source URL link manually to the `remotes` field " *
        "in `makedocs` or install the package using `Pkg.develop()``."
)
```
etc